Testing/functional/tests/cypress/integration/e2e/timeline/comments.js

```diff
@@ -2,30 +2,47 @@ const { AuthMethod } = require("../../../support/constants");
 
 describe("Comments Disable", () => {
     beforeEach(() => {
-        cy.enableModules(["Laboratory"]);
+        cy.configureSettings({
+            datasets: [
+                {
+                    name: "covid19TestResult",
+                    enabled: true,
+                },
+            ],
+        });
         cy.login(
             Cypress.env("keycloak.username"),
             Cypress.env("keycloak.password"),
             AuthMethod.KeyCloak
         );
     });
     it("Comments Disable", () => {
-        cy.enableModules(["Laboratory"]);
+        cy.configureSettings({
+            datasets: [
+                {
+                    name: "covid19TestResult",
+                    enabled: true,
+                },
+            ],
+        });
         cy.get("[data-testid=addCommentTextArea]").should("not.exist");
         cy.get("[data-testid=postCommentBtn]").should("not.exist");
     });
 });
 
 describe("Comments Enable", () => {
     beforeEach(() => {
-        cy.configureSettings(
-            {
-                timeline: {
-                    comment: true,
-                },
+        cy.configureSettings({
+            timeline: {
+                comment: true,
             },
-            "Laboratory"
-        );
+            datasets: [
+                {
+                    name: "clinicalDocument",
```

Comment:
why the change to 'clinicalDocument' from 'laboratory'?

Reply:
Fixed